examples/replay/fill_quarter_degree.py

```diff
@@ -0,0 +1,71 @@
+import os
+import subprocess
+
+from ufs2arco import FV3Dataset, Timer
+from replay_mover import ReplayMoverQuarterDegree
+
+
+def submit_slurm_filler(job_id, mover):
+
+    the_code = \
+        f"from replay_mover import ReplayMoverQuarterDegree\n"+\
+        f"from replay_filler import ReplayFiller\n"+\
+        f"mover = ReplayMoverQuarterDegree(\n"+\
+        f"    n_jobs={mover.n_jobs},\n"+\
+        f"    n_cycles={mover.n_cycles},\n"+\
+        f"    config_filename='{mover.config_filename}',\n"+\
+        f"    storage_options={mover.storage_options},\n"+\
+        f"    main_cache_path='{mover.main_cache_path}',\n"+\
+        f")\n"+\
+        f"filler = ReplayFiller(mover=mover)\n"+\
+        f"filler.run({job_id})"
+
+    slurm_dir = "slurm/replay-fill-0.25-degree"
+    txt = "#!/bin/bash\n\n" +\
+        f"#SBATCH -J rfqd{job_id:03d}\n"+\
+        f"#SBATCH -o {slurm_dir}/{job_id:03d}.%j.out\n"+\
+        f"#SBATCH -e {slurm_dir}/{job_id:03d}.%j.err\n"+\
+        f"#SBATCH --nodes=1\n"+\
+        f"#SBATCH --ntasks=1\n"+\
+        f"#SBATCH --cpus-per-task=30\n"+\
+        f"#SBATCH --partition=compute\n"+\
+        f"#SBATCH -t 120:00:00\n\n"+\
+        f"source /contrib/Tim.Smith/miniconda3/etc/profile.d/conda.sh\n"+\
```

Comment:
Would it be possible to replace "Tim.Smith" with `os.getlogin()` here and other places down below.

Reply:
Discussed in #12